core/dbt/parser/models.py

```diff
@@ -26,46 +27,106 @@ def render_update(
+                else:
+                    # look for false positive configs
+                    for c in experimentally_parsed['configs']:
+                        if c not in config._config_calls:
+                            value += ["02_false_positive_config_value"]
+                            break
+
+                    # look for missed configs
+                    for c in config._config_calls:
+                        if c not in experimentally_parsed['configs']:
+                            value += ["03_missed_config_value"]
+                            break
 
-                # this uses the updated config to set all the right things in the node
-                # if there are hooks present, it WILL render jinja. Will need to change
-                # when we support hooks
-                self.update_parsed_node(node, config)
+                    # look for false positive sources
+                    for c in experimentally_parsed['sources']:
+                        if c not in node.sources:
+                            value += ["04_false_positive_source_value"]
+                            break
 
-                # udpate the unrendered config with values from the file
-                # values from yaml files are in there already
-                node.unrendered_config.update(dict(res['configs']))
+                    # look for missed sources
+                    for c in node.sources:
+                        if c not in experimentally_parsed['configs']:
+                            value += ["05_missed_source_value"]
+                            break
 
-                # set refs, sources, and configs on the node object
-                node.refs = node.refs + res['refs']
-                for sourcev in res['sources']:
-                    # TODO change extractor to match type here
-                    node.sources.append([sourcev[0], sourcev[1]])
-                for configv in res['configs']:
-                    node.config[configv[0]] = configv[1]
+                    # look for false positive refs
+                    for c in experimentally_parsed['refs']:
+                        if c not in node.refs:
+                            value += ["06_false_positive_ref_value"]
+                            break
 
-                self.manifest._parsing_info.static_analysis_parsed_path_count += 1
+                    # look for missed refs
+                    for c in node.refs:
+                        if c not in experimentally_parsed['refs']:
+                            value += ["07_missed_ref_value"]
+                            break
 
-            # exception was thrown by dbt jinja extractor meaning it can't
-            # handle this source. fall back to python jinja rendering.
-            except ExtractionError:
-                super().render_update(node, config)
+                    # dedup values
+                    value = list(set(value))
+
```

Comment:
If `value` is empty here at the end, what do you think of sticking in some kind of `"00_success"` or `"00_exact_match"`?

Reply:
yes we should totally do that. added.